meta-resin-common/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bb

```diff
@@ -25,7 +25,7 @@ do_configure[noexec] = "1"
 
 do_compile() {
     mkdir -p kernel_modules_headers
-    ${S}/gen_mod_headers ./kernel_modules_headers ${STAGING_KERNEL_DIR} ${DEPLOY_DIR_IMAGE} ${ARCH} ${TARGET_PREFIX} "${CC}"
```

Comment:
I'm not sure why is this the case because we do deploy the config in deploy dir too.